test/integration/framework/etcd_utils.go

```diff
@@ -30,9 +30,8 @@ import (
 // If you need to start an etcd instance by hand, you also need to insert a key
 // for this check to pass (*any* key will do, eg:
 //curl -L http://127.0.0.1:2379/v2/keys/message -XPUT -d value="Hello world").
-func init() {
-	RequireEtcd()
-}
+
```

Comment:
@wojtek-t replied below... removing `init()` so that `t.Skip(..)` works.  The broader reason to remove it is in the comment below.